doc/Quality_white_wine_predictor.md

```diff
@@ -0,0 +1,245 @@
+Quality white wine predictor
```

Comment:
Since we already have the doc/Quality_white_wine_predictor.Rmd file, do we need this one as well?